libs/wingsdk/src/cloud/api.ts

```diff
@@ -270,22 +275,13 @@ export enum HttpMethod {
   /** Delete */
   DELETE = "DELETE",
   /** Connect */
-  CONNECT = "Connect",
+  CONNECT = "CONNECT",
   /** Options */
   OPTIONS = "OPTIONS",
   /** Patch */
   PATCH = "PATCH",
 }
 
-// /**
-//  * Json type representation.
-//  */
-// // TODO: this should come from a shared package
```

Comment:
Delete?

Reply:
It was commented out, I figured we can always add it back later if we need it